### PR TITLE
Update links in README

### PR DESCRIPTION
**Pull request by outcatcher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ OpenTelekomCloud CCE driver for Rancher
 
 1. Go to `Cluster Drivers`
 2. Click `Add cluster driver`
-3. Set download URL: https://csm-assets.obs.eu-de.otc.t-systems.com/kontainer-engine-driver-otc-0.2.1-linux-amd64.tgz
-4. Set custom UI URL: https://csm-assets.obs.eu-de.otc.t-systems.com/ui/component.js
+3. Set download URL: https://otc-rancher.obs.eu-de.otc.t-systems.com/cluster/driver/latest/kontainer-engine-driver-otccce_linux_amd64.tar.gz
```
Comment (kucerakk) [inline README.md]: shouldnt we use the same links as here:
https://github.com/opentelekomcloud/kontainer-engine-driver-otc/pull/31/files

Comment (kucerakk) [reply README.md]: `https://github.com/opentelekomcloud/kontainer-engine-driver-otc/releases/download/VERSION/kontainer-engine-driver-otc-VERSION-linux-amd64.tgz`

Comment (outcatcher) [reply README.md]: I suppose we should have `https://otc-rancher.obs.eu-de.otc.t-systems.com/cluster/driver/latest/kontainer-engine-driver-otccce_linux_amd64.tar.gz` and `https://otc-rancher.obs.eu-de.otc.t-systems.com/cluster/ui/latest/component.js` in both repos in this case

Comment (kucerakk) [reply README.md]: ok, then we should keep these link in sync with both repos...the driver and the UI
furthermore we have to make sure the releases are uploaded to OBS

**Review by kucerakk — APPROVED** [review]

**Pull request merged**